source/_integrations/sense.markdown

```diff
@@ -51,3 +51,5 @@ Sensors are added for both usage and production with the following names:
 - **Daily/Weekly/Monthly Usage/Production**: Daily/Weekly/Monthly power usage/production in kWh. Updated every 5 minutes.
 
 Binary sensors are created for each of the devices detected by your Sense monitor to show their power state.
+
+In version 0.107 and later, usage sensors are created for each of the devices detected by your Sense monitor to show their power usage in Watts.
```

Comment:
We don't add changelogs into our documentation.

Please adjust the documentation to reflect the latest codebase state.